pkg/controller.v1alpha1/xgboost/pod.go

```diff
@@ -15,16 +15,46 @@
 package xgboost
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/kubeflow/common/util/k8sutil"
+
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	common "github.com/kubeflow/common/operator/v1"
+	"github.com/kubeflow/tf-operator/pkg/common/jobcontroller"
 	"github.com/kubeflow/xgboost-operator/pkg/apis/xgboost/v1alpha1"
 	log "github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	pylogger "github.com/kubeflow/tf-operator/pkg/logger"
+
+)
+
+const (
```

Comment:
Reuse these from common: https://github.com/kubeflow/common/blob/master/job_controller/pod.go#L40-L51

Reply:
it seems these constant are not exported ? I have fixed other parts based on your comments. thanks Terry.